tests/qchem/test_structure.py

```diff
@@ -0,0 +1,313 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the functions of the structure module.
+"""
+import os
+import pytest
+import pennylane as qml
+from pennylane import qchem
+from pennylane import numpy as np
+from pennylane.templates.subroutines import UCCSD
+
+
+ref_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)), "test_ref_files")
+
+
+def test_reading_xyz_file(tmpdir):
+    r"""Test reading of the generated file 'structure.xyz'"""
+
+    ref_symbols = ["C", "C", "N", "H", "H", "H", "H", "H"]
+    ref_coords = np.array(
```

Comment:
Should we use 2D arrays for coordinates?

Reply:
Both 2D and 1D are supported. I intentionally did not change the test structures.